torch_xla/csrc/xla_graph_executor.cpp

```diff
@@ -590,6 +593,15 @@ XLAGraphExecutor::ExecuteComputationWithBarrier(
     torch::lazy::BackendDataPtr handle =
         WrapXlaData(xla::ComputationClient::Get()->CreateDataPlaceholder(
```

Comment:
If it's SPMD virtual device, then we should always use PjRtShardedData handle.

Reply:
hmm, is the logic below to call `WrapDataShards` not enough? This code path is shared between spmd and non-spmd code path.